zulipterminal/model.py

```diff
@@ -373,6 +389,8 @@ def get_topics_in_stream(self, stream_list: Iterable[int]) -> str:
                 self.index['topics'][stream_id] = [topic['name'] for
                                                    topic in response['topics']]
             else:
+                if response['result'] == 'error':
+                    self.controller.view.set_footer_text(response['msg'], 3)
```

Comment:
Given the apparent similarity in all this error handling, please use a new method for this.

Putting this in a function should make testing simple?